src/routes/guides/building-a-data-source/+page.svelte

```diff
@@ -0,0 +1,499 @@
+    classes from the final definition we created in the <A href={GUIDES.BUILD_DATA_DEF.href}
+      >{GUIDES.BUILD_DATA_DEF.title}</A
+    > guide. Note that we also need to add the necessary imports.
+  </p>
+  <p>
+    If you're building a data source for another definition, it's possible it was created using
+    pydantic models, in which you can retrieve those from within the <em>/src</em> directory in the
+    definitions repository. If they are not available you will have to build them yourself. In that
+    case the same guide can be handy, as well as the official pydantic documentation and the
+    <A href="https://fastapi.tiangolo.com/tutorial/body/">Request Body</A> section of the FastAPI docs.
+  </p>
+  <p>The <em>app/models.py</em> file would after these changes look like this:</p>
+
+  <Code lang={python}>
+    {`
+from typing import List, Optional
+
+from pydantic import BaseModel, Field, constr
+from stringcase import camelcase
+
+
+class CamelCaseModel(BaseModel):
+    class Config:
+        alias_generator = camelcase
+        allow_population_by_field_name = True
+
+
+class HealthResponse(CamelCaseModel):
+    status: bool
+
+
+class BasicCountryInfoRequest(CamelCaseModel):
+    code: str = Field(
+        ...,
+        title="Code",
+        description="ISO 3166-1 alpha-2 code for the country",
+        example="FI",
+        min_length=2,
+        max_length=2,
+    )
+
+
+class Capital(CamelCaseModel):
+    name: str = Field(
+        ...,
+        title="Name",
+        description="The name of the capital of the Country",
+        example="Helsinki",
+    )
+    lat: float = Field(
+        ...,
+        title="Latitude",
+        description="The latitude coordinate of the Capital",
+        ge=-90.0,
+        le=90.0,
+        example=60.170833,
+    )
+    lon: float = Field(
+        ...,
+        title="Longitude",
+        description="The longitude coordinate of the Capital",
+        ge=-180.0,
+        le=180.0,
+        example=24.9375,
+    )
+
+
+class BasicCountryInfoResponse(CamelCaseModel):
+    code: str = Field(
+        ...,
+        title="Code",
+        description="ISO 3166-1 alpha-2 code for the country",
+        example="FI",
+        min_length=2,
+        max_length=2,
+    )
+    name: str = Field(
+        ...,
+        title="Name",
+        description="The name of the country",
+        example="Finland",
+    )
+    area: float = Field(
+        ...,
+        title="Area",
+        description="The area of the country in km^2",
+        example=338455,
+    )
+    languages: List[constr(min_length=2, max_length=2)] = Field(
+        ...,
+        title="Official languages",
+        description="ISO 639-1 language codes for the official languages",
+        example=["fi", "sv"],
+    )
+    capital: Optional[Capital] = Field(
+        None,
+        title="Capital",
+        description="The capital of the country, legislative if multiple",
+    )
+`}
+  </Code>
+  <SectionTitle
```

Comment:
There's couple of these SectionTitles that are split in this odd way on ~4 rows. Not that it matters much, but might be nice to clean up.